specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/examples/ApiManagementGatewayGetKubernetesManagedGateway.json

```diff
@@ -0,0 +1,53 @@
+{
+  "parameters": {
+    "gatewayName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2024-10-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ApiManagement/gateway/example-gateway",
+        "name": "example-gateway",
+        "type": "Microsoft.ApiManagement/gateway",
+        "tags": {
+          "owner": "v-aswmoh",
+          "ReleaseName": "Z3"
+        },
+        "location": "East US",
+        "etag": "AAAAAAAWN/4=",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "targetProvisioningState": "",
+          "createdAtUtc": "2021-06-16T09:40:00.9453556Z",
+          "frontend": {
+            "defaultHostname": "example-gateway.eastus.gateway.azure-api.net"
+          },
+          "backend": {
+            "subnet": {
+              "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworks/vn1/subnets/sn1"
+            }
+          },
+          "configurationApi": {
+            "hostname": "example-gateway.eastus.configuration.gateway.azure-api.net"
+          }
```

Comment:
Not sure if we should keep showing this, if we can let's drop it

Reply:
Is there a good rule of thumb for which APIs should return it and which should not?